textattack/commands/attack/attack_args_helpers.py

```diff
@@ -427,13 +428,34 @@ def parse_logger_from_args(args):
 
     # if "--log-to-file" specified in terminal command, then save it to a txt file
     if args.log_to_file:
-        # Output file.
-        outfile_name = "attack-{}.txt".format(out_time)
+        # if "--txt-filename" specified, use that as filename
+        if args.txt_filename:
+            outfile_name = "{}.txt".format(args.txt_filename)
+        else:
+            if args.recipe:
+                outfile_name = "TXT_AttackResult_{}_{}_{}-{}-{}-{}-{}.txt".format(
```

Comment:
I don't think we need the "TXT_AttackResult_" part. Same goes for "CSV_AttackResult_" below. Also use [f-strings](https://www.geeksforgeeks.org/formatted-string-literals-f-strings-python/)?